babel.config.js

```diff
@@ -0,0 +1,4 @@
+{
+    "presets"; ["@babel/preset-env"]
```

Comment:
I'm not too familiar with babel so I searched up the [config docs](https://babeljs.io/docs/babel-preset-env) to double check. I think it should be a colon after presets but if it's working with the semi-colon perhaps not? In any case wanted to point it out in case it was an error.

Reply:
I originally had it as a colon, and it presented an error until I did the semicolon. I was also surprised!